cf-java-monitoring-custom-metrics-clients/cf-custom-metrics-clients-spring-boot/pom.xml

```diff
@@ -52,6 +52,20 @@
             <artifactId>spring-boot-starter-actuator</artifactId>
             <version>${spring.boot.version}</version>
             <scope>provided</scope>
+            <exclusions>
```

Comment:
I see how this circumvents security issues. Almost any library user will have these dependencies. It eases maintenance, but hides when a version update is required from a library user. It will lead to issues to upgrade rather than failing security scans.